### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by qrispi** [qrispi/overlook#3]
- Create a branch for accessibility.
- Use this branch to make your dashboard as accessible as possible.
- Push this branch up to GH. You can merge the changes into main but do not delete this branch.
- This branch should not have a login page so that during the live eval, we can run the Lighthouse audit and check tabbability of your dashboard (without the login page).

- Accessibility audits should be at 100% for the dashboard.
- A user should be able to interact with all functionality of your application by tabbing through it, no use of the trackpad
- ARIA attributes should be utilized for any UI elements that are not understood by the screen reader
